airflow/executors/executor_loader.py

```diff
@@ -65,18 +65,23 @@ class ExecutorLoader:
         DEBUG_EXECUTOR: "airflow.executors.debug_executor.DebugExecutor",
     }
 
+    @classmethod
+    def get_default_executor_name(cls) -> str:
+        """Returns the default executor name from Airflow configuration
```

Comment:
```suggestion
        """Returns the default executor name from Airflow configuration.
```

Reply:
If you view the rest of the module, all the docs use the plural verb form (`Loads the executor...`, `Creates a new instance of...`) so I will keep that form here. But I will add the missing period!